plugin/trino-spooling-filesystem/src/main/java/io/trino/spooling/filesystem/FileSystemSpoolingConfig.java

```diff
@@ -28,6 +29,9 @@ public class FileSystemSpoolingConfig
     private String location;
     private Duration ttl = new Duration(2, HOURS);
     private boolean encryptionEnabled = true;
+    private boolean pruningEnabled = true;
+    private Duration pruningInterval = new Duration(5, MINUTES);
+    private long pruningBatchSize = 250;
```

Comment:
arbitrary? ****

Reply:
If S3 supports multi-file delete requests - then the bigger request the cheaper. Probably does not matter much. How many delete requests do we expect in a cluster per hour?

Reply:
We batch our multi delete calls to 250 objects so I kept the same value

Reply:
https://github.com/trinodb/trino/blob/master/lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3FileSystem.java#L143